### ISSUE TITLE
[java] JUnitTestsShouldIncludeAssert should support Junit 5

### ISSUE THREAD

**Issue by oowekyala** [pmd/pmd#3341]
**Affects PMD Version:** 6.35.0

**Rule:** [JUnitTestsShouldIncludeAssert](https://pmd.github.io/latest/pmd_rules_java_bestpractices.html#junittestsshouldincludeassert)

**Description:** Assertion methods are in a class named `Assertions` in JUnit 5. The rule doesn't handle that for now.

**Code Sample demonstrating the issue:**

```java
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.Assertions;
class Test {
  @Test 
  void someTest() {
    Assertions.assertEquals("a", "b");
  }
}
```

**Expected outcome:** No violation

PMD reports a violation at line ..., but that's wrong. That's a false positive.

**Running PMD through:** any
